app/src/test/java/edu/uc/group/rankine/EditingSetTest.kt

```diff
@@ -1,7 +1,6 @@
 package edu.uc.group.rankine
 
 import edu.uc.group.rankine.dto.ElementObject
-import edu.uc.group.rankine.dto.FieldObject
```

Comment:
Removed broken import for DTO that doesn't actually exist.

Reply:
FieldObject and ElementObject sound a bit ambiguous.  :)